crates/module-info-parser/src/effects.rs

```diff
@@ -22,7 +22,8 @@ use walrus::Module;
 
 use std::path::Path;
 
-const HOST_IMPORT_NAMESPACE: &str = "host";
+const HOST_IMPORT_NAMESPACE_V0: &str = "host";
+const HOST_IMPORT_NAMESPACE_PREFIX: &str = "__marine_host_api_v";
```

Comment:
Let's put TODO here to create a shared crate that contains these definitions and could be used by both `Marine` and `marine-rs-sdk`.